cmd/iacinventorycmd/iacinventorycmd.go

```diff
@@ -0,0 +1,53 @@
+package iacinventorycmd
+
+import (
+	"bytes"
+
+	"github.com/soluble-ai/soluble-cli/pkg/iacinventory"
+	"github.com/soluble-ai/soluble-cli/pkg/options"
+	"github.com/spf13/cobra"
+)
+
+func Command() *cobra.Command {
+	var username string
+	var oauthToken string
+	var submit bool
+	opts := options.PrintClientOpts{}
+	c := &cobra.Command{
+		Use:   "iac-inventory",
+		Short: "Run an Infrastructure-as-code inventorier on repositories",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// For now, we only support Github.
+			scanner := iacinventory.New(&iacinventory.GithubScanner{
+				User:       username,
+				OauthToken: oauthToken,
+			})
+			result, err := scanner.Run()
+			if err != nil {
+				return err
+			}
+			opts.PrintResult(result)
+			var buf bytes.Buffer
+			opts.OutputFormat = "json" // is there a cleaner way to do this?
```

Comment:
Yeah, just json.Marshal().  opts.PrintResult is for printing results out to the user.